pkg/components/exporter/exporter.go

```diff
@@ -49,6 +49,8 @@ const (
 	ServiceName   = prefix + "svc"
 
 	StableImage = "quay.io/sustainable_computing_io/kepler:release-0.5.4"
+
+	RedfishSecretName = "redfish"
```

Comment:
Where are we creating the secret ? 🤔

Reply:
If the redfish secret is optional, how about we allow 
```
exporter:
  redfish:
    secretRef: 
```

Also, why are we using secret instead of say configmap?